File: practicas/prt42/figura.java
package practicas.prt42;

public abstract class figura {
    private String color;

    public figura(String color) {
        this.color = color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public String getColor() {
        return color;
    }

    public abstract double area();
    public abstract double perimetro();

    @Override
    public String toString() {
        return "Color: " + color;
    }
}

File: practicas/prt42/figura.py
class Figura:
    def __init__(self, color):
        self._color = color

    def set_color(self, color):
        self._color = color

    def get_color(self):
        return self._color

    def area(self):
        raise NotImplementedError("Subclasses must implement this method.")

    def perimetro(self):
        raise NotImplementedError("Subclasses must implement this method.")

    def __str__(self):
        return f"Color: {self._color}"
